bridges/FindACrewBridge.php

```diff
@@ -62,11 +62,17 @@ public function collectData() {
 		foreach ($annonces as $annonce) {
 			$item = array();
 
-			$img = parent::getURI() . $annonce->find('.lst-pic img', 0)->getAttribute('src');
+			$link = parent::getURI() . $annonce->find('.lst-ctrls a', 0)->href;
+			$htmlDetail = getSimpleHTMLDOMCached($link . '?mdl=2'); // add ?mdl=2 for xhr content not full html page
```

Comment:
Thanks for the PR!

One suggestion regarding absolute URLs. Instead of inserting `parent::getURI()` manually, you can simply utilize the `defaultLinkTo` function: https://github.com/RSS-Bridge/rss-bridge/wiki/defaultLinkTo

In your case, this should work

```php
$htmlDetail = defaultLinkTo($htmlDetail, $this->getURI());
```

This can be used instead of these lines below

```php
$img = parent::getURI() . $htmlDetail->find('img.img-responsive', 0)->getAttribute('src');
...
$content = str_replace('src="/', 'src="' . parent::getURI() . '/', $content);
$content = str_replace('href="/', 'href="' . parent::getURI() . '/', $content);
```

I haven't checked the remaining code, but it looks like it can benefit from this as well.